src/app/components/colors/colors.component.css

```diff
@@ -0,0 +1,19 @@
+/* Styling for colour div */
+.colour {
+    width: 30px;
+    height: 30px;
+    border-radius: 50%;
+}
+
+/* Tick mark CSS */
+.fa-thin {
```

Comment:
add your own property name, else it might override the existing style from vendor